content/slides/en.yaml

```diff
@@ -78,57 +111,121 @@ levels:
             code: |-
                 {print} Welcome to your own rock scissors paper!
                 {ask} What will you choose?
+            notes: |-
+                Show that the computer now asks you a question. Fill in an answer and also show that once you fill in the answer, nothing happens.
+                Make your students aware that the computer will only follow your instructions, it will not do anything by itself. This means that if you want the computer to print the answer, you have to give that instruction.
         5:
             header: '{echo}'
             text: |-
-                As you could see in the previous example. If you use the `{ask}` command, you can ask a question.
-                The answer, however doesn't show in the output screen.
-
-                If you do want the answer to show, you can use the final command of this level: `{echo}`.
-                `{echo}` is used to repeat the answer of an `{ask}`.
+                To show the answer in the output screen, you can use: `{echo}`.
+                `{echo}` is never used by itself, always with an `{ask}`.
             code: |-
                 {print} Hello!
                 {ask} What is your name?
                 {echo} hello
+            notes: |-
+                To make the computer repeat the answer back to you, you use the `echo` command. Point out that the answer you've typed in will appear at the end of the sentence.
         6:
             header: Programming!
-            text: |-
-                With `{print}`, `{ask}` and `{echo}` you can already create a little story.
-                This is a good time to try the Parrot, Story and Rock, Paper Scissors adventures.
+            text: Here you can find a list of the adventures.
             editor: /hedy/1
+            notes: |-
+                Before we start, show the students the adventures. Don't forget to point out the exercises. This is where the students will have to read what to do exactly in an adventure.
         7:
-            header: Programming!
-            text: Adventures are shown in tabs.
-            editor: /hedy/1
-        8:
             header: Programming!
             text: The yellow arrow buttons can be used to copy examples.
             editor: /hedy#print_command
+            notes: |-
+                Lastly show the students that they can copy the example code into their programming field by pressing the yellow button.
+        8:
+            header: Let's get started!
+            text: |-
+                This is a good time to try the Parrot, Haunted House, Story and Rock, Paper Scissors adventures.
+            editor: /hedy/1
+            notes: |-
+                These are the first adventures to start with. Once (most of) the students are done, you can introduce the drawing turtle and the `{play}` command.
```

Comment:
Whe don't mention the debug slide here. When is good moment to show debugging?

Reply:
I thought about doing the debugger first as well, but then I thought the instructions would take way too long and it would be better to first get the students to work and later show the debugger. They don't really need it in the first level anyway because the programs are only a couple of lines.